Application/EdFi.Ods.Features/OpenApiMetadata/Factories/OpenApiMetadataPathsFactory.cs

```diff
@@ -406,7 +422,7 @@ private Operation CreateDeletesOperation(OpenApiMetadataPathsResource openApiMet
                     new Response
                     {
                         description =
-                            "Gone. An attempt to connect to the database for the snapshot specified by the Snapshot-Identifier header was unsuccessful (indicating the snapshot may have been removed)."
+                                                        "Gone. An attempt to connect to the database for the snapshot specified by the Snapshot-Identifier header was unsuccessful (indicating the snapshot may have been removed)."
```

Comment:
Whitespace change is a bit odd here. Can we reformat (or revert) this? Was this formatted this way by Resharper?

Reply:
I thought that it was reformatted by Resharper, but I triggered the formatting again and I couldn't reproduce it, probably I inadvertently changed it.
Reverted it.